content/20.treatments.md

```diff
@@ -559,7 +560,7 @@ Concomitant administration of vitamin D with pharmaceuticals such as aspirin [@c
 
 While there is increasing evidence that vitamin D status is linked to COVID-19 outcomes, the effectiveness of its supplementation remains open for debate.
 Once again, supplementation of vitamin D and maintaining a healthy diet for optimum vitamin D status is unlikely to carry major health risks while the possible link between vitamin D status and COVID-19 is investigated.
-However, pursuing to elevate vitamin D levels through sunlight exposure does carry additional risks, as many densely populated cities around the world are utilizing ‘stay in place’ orders to enforce social distancing guidelines.
+However, pursuing to elevate vitamin D levels through sunlight exposure does carry additional risks, as many densely populated cities and countries around the world are sporadically utilizing ‘stay in place’ orders to enforce social distancing guidelines as cases have began to rise as winter approaches.
```

Comment:
```suggestion
However, pursuing to elevate vitamin D levels through sunlight exposure does carry additional risks, as many densely populated cities and countries around the world are sporadically utilizing ‘stay in place’ orders to enforce social distancing guidelines, especially with cases rising again as winter 2020 approaches.
```
I am not totally sure what tense to use here! Since it's in the future now I think this is OK?

Reply:
I think this is ok.